stdlib/public/Concurrency/Task.swift

```diff
@@ -194,18 +194,29 @@ extension Task {
     }
   }
 
-  /// Indicates that the task should stop running.
+  /// Marks this task as cancelled.
   ///
   /// Task cancellation is cooperative:
   /// a task that supports cancellation
-  /// checks whether it has been canceled at various points during its work.
+  /// checks whether it has been canceled at various points during its work,
+  /// and may either return early or throw an error in order to
+  /// finish running earlier than it would if it wasn't cancelled.
+  ///
+  /// Calling this method only has the effect of setting the cancelled
+  /// flag on the task (or its child tasks), and unless the tasks itself
+  /// react on this flag, there is no observable runtime behavior change
+  /// of the tasks. Specifically, tasks are never "interrupted and stopped"
+  /// forcefully, and are always allowed to execute code until returning or throwing/
```

Comment:
With just the spelling change:

```suggestion
  /// checks whether it has been canceled at various points during its work,
  /// and may either return early or throw an error in order to
  /// finish running earlier than it would if it wasn't canceled.
  ///
  /// Calling this method only has the effect of setting the canceled
  /// flag on the task (or its child tasks), and unless the tasks itself
  /// react on this flag, there is no observable runtime behavior change
  /// of the tasks. Specifically, tasks are never forcefully interrupted and stopped,
  /// and are always allowed to execute code until they returning or throwing.
```

With more revision:


```suggestion
  /// checks whether it has been canceled at various points during its work.
  /// You determine what action a canceled task takes,
  /// and whether cancellation does anything to that task.
  /// The task can return early or throw an error
  /// to stop running early.
  ///
  /// Calling this method has only one effect:
  /// setting the canceled flag on the task or its child tasks.
  /// Unless the tasks react to this flag,
  /// cancellation doesn't change the behavior of the tasks.
  /// Specifically, tasks are never forcefully interrupted or stopped,
  /// and are always allowed to execute code until they return or throw an error.
```